src/transaction.rs

```diff
@@ -133,3 +134,86 @@ pub trait TransactionsSink<H: BlockHash>: Send + Sync {
 
 	fn transaction_monitor(&self) -> Option<&dyn TransactionMonitor<H>>;
 }
+
+#[derive(Subcommand)]
+/// Send transactions to the node using different scenarios.
+pub enum SendingScenario {
```

Comment:
maybe this could be moved to the dedicated file?

Reply:
I moved this back to the cli module. I created a new module called `scenario`, where I developed some logic related to the idea of a `scenario`. I kept the `SendingScenario` in the `cli` though (seeing it as a subcommand made me think that's its place, but lmk if you have another idea).

Reply:
>  I kept the SendingScenario in the cli though

Spoke to soon: [cf898d4](https://github.com/michalkucharczyk/tx-test-tool/pull/22/commits/cf898d4a74adc76c20bdcf86d5b428657fd7ea1f)